CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# dbt_hubspot_source v0.14.0
+[PR #122](https://github.com/fivetran/dbt_hubspot_source/pull/122) includes the following updates:
+
+## 🚨 Breaking Change 🚨 
+- When including a passthrough `property_hs_*` column, you now have the option to include the corresponding, human-readable label in the staging models. 
+  - See the [Adding property label section](https://github.com/fivetran/dbt_hubspot_source#adding-property-label) of the README for instructions on how to enable this feature! 
+  - This update applies to models:
+    - `stg_hubspot__company`
+    - `stg_hubspot__contact`
+    - `stg_hubspot__deal`
+    - `stg_hubspot__ticket`
```

Comment:
Is this really a breaking change? This seems more like a feature. Thoughts?

Reply:
Ahh okay I see what you mean. I would actually request this be folded into the feature updates and then provide a blurb that leveraging this new feature you will require the property tables and if you do not have them then you will need to disable the tables and you cannot leverage this feature.

When I see breaking change I think something from the previous build is now different and can change my downstream transformations. Reading through this I don't get the impression this is a 🚨 Breaking 🚨 change, but rather a feature that will result in a 0.X.0 version bump.